docs/user-guide/interfaces/cli/developing-rugs.md

```diff
@@ -172,6 +166,18 @@ be faster.
 As you can see from the last line of output, all of the test scenarios
 passed.
 
+To run a specific named test:
+
+```console
+$ rug test "Whatever Test Secanrio"
+```
+
+To run all scenarios from a .rt file:
+
+```console
+$ rug test MyRugTestFilename
+```
+
```

Comment:
Running only one test or all tests of the certain test file is actually not working with Rug 0.13.0 as the Gherkin support doesn't provide that (yet?). We should consider removing this here.